gradle/sonar.gradle

```diff
@@ -0,0 +1,18 @@
+apply plugin: 'org.sonarqube'
+
+def analysisMode = 'publish'
+//TODO(kzaikin): use preview mode in pull requests
+//if (new Boolean(System.env.TRAVIS_PULL_REQUEST)) {
+//    analysisMode = 'preview'
+//}
+
+sonarqube {
+    properties {
+        property 'sonar.analysis.mode', analysisMode
+        property 'sonar.host.url', System.env.SONAR_HOST_URL
+        property 'sonar.login', '250cb5c72d59d12620f1733fc7f397850d76ade0'
```

Comment:
@KengoTODA please take a look at temporary analyzed project
https://sonarcloud.io/dashboard?id=kzaikin.com.github.spotbugs.spotbugs

Looks like we must define sonar organization to analyze project. It's a new feature for sonarcloud.
Did you create a Spotbugs organization at sonarqube? Could you tell me its name for I to specify it here?